ingestion/src/main.rs

```diff
@@ -71,8 +75,28 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     let cancel_token = CancellationToken::new();
     tokio::spawn(util::signal_catcher(cancel_token.clone()));
 
+    // Set up prometheus metrics exporter
+    PrometheusBuilder::new()
+        .set_buckets_for_metric(
+            Matcher::Full("http_requests_duration_seconds".to_string()),
+            &[
+                0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0,
+            ],
+        )
+        .expect("Failed to set metric buckets")
+        .install()
+        .expect("Failed to set up metrics exporter");
+
+    // Register metrics so they're guaranteed to show in exporter output
+    let _ = metrics::histogram!("http_requests_duration_seconds");
```

Comment:
Suggest to declare metric names as string constants and replace existing occurrences with those constants. This reduces the risk of accidental typos slipping through undetected.

Reply:
Excellent suggestion! Done